docs/payment-methods/GOOGLE_PAY.md

```diff
@@ -0,0 +1,140 @@
+| `setCountryCode`                   | Sets the ISO 3166-1 alpha-2 country code where the transaction is processed.                                                                      |
+| `setAllowedAuthMethods`            | Sets the supported authentication methods.                                                                                                        |
+| `setAllowedCardNetworks`           | Sets the allowed card networks. The allowed networks are automatically configured based on your account settings, but you can override them here. |
+| `setAllowPrepaidCards`             | Set to `true` if you support prepaid cards.                                                                                                       |
+| `setAllowCreditCards`              | Set to `true` if you support credit cards.                                                                                                        |
+| `setAssuranceDetailsRequired`      | Set to `true` if you want to request assurance details.                                                                                           |
+| `setEmailRequired`                 | Set to `true` if an email address is required.                                                                                                    |
+| `setExistingPaymentMethodRequired` | Set to `true` if an existing payment method is required.                                                                                          |
+| `setShippingAddressRequired`       | Set to `true` if a shipping address is required.                                                                                                  |
+| `setShippingAddressParameters`     | Allows to configure the shipping address parameters.                                                                                              |
+| `setBillingAddressRequired`        | Set to `true` if a billing address is required.                                                                                                   |
+| `setBillingAddressParameters`      | Allows to configure the billing address parameters.                                                                                               |
+| `setTotalPriceStatus`              | Sets the status of the total price used.                                                                                                          |
+| `setGooglePayButtonStyling`        | Allows to configure the styling of the Google Pay button.                                                                                         |
+
+## Migrating to 5.8.0+
+It is not necessary to migrate, but 5.8.0 introduced a simplified integration for Google Pay. This new integration among others gets rid of the deprecated `onActivityResult` and includes the Google Pay button. Follow the steps below to migrate from previous 5.x.x versions to 5.8.0:
+
+1. Remove deprecated Activity Result code
```

Comment:
This line could be bold or somehow marked up to make it easier to read. Applies to line 118 and 138 as well.